### PR TITLE
Fixup vibe.d test project

### PR DESCRIPTION
**Pull request by s-ludwig**
The existing test failed to pass any of the tests. In addition to making that work, this PR also uses more idiomatic D code and implements the remaining tests (updates/fortunes).

### CODE REVIEW

**Comment by whitebyte** [comment]
I also think compiler vesion (https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/toolset/setup/linux/languages/dlang.sh#L11) should be updated as well, or replaced with GDC/LDC,

**Comment by s-ludwig** [comment]
Usernames (e.g. --client-user, --runner-user, and --database-user) are required!

Any idea about that error? I don't think I have changed anything related to that and I'm a bit lost where to start looking for a fix

> I also think compiler vesion (...) should be updated as well, or replaced with GDC/LDC,

True, I'd leave that to a separate PR, though. Another thing is that a MySQL, PostgreSQL and/or Redis version would be nice to have.

**Comment by s-ludwig** [comment]
>     Usernames (e.g. --client-user, --runner-user, and --database-user) are required!
Any idea about that error? I don't think I have changed anything related to that and I'm a bit lost where to start looking for a fix

BTW, the test is successful for me locally (testing in a vagrant environment).

**Comment by s-ludwig** [comment]
This includes/supercedes #1723.

File: frameworks/D/vibed/benchmark_config.json [hunk 1]
```diff
@@ -7,18 +7,20 @@
...
       "framework": "vibed",
       "language": "D",
-      "orm": "Raw",
+      "orm": "Micro",
       "platform": "D",
       "webserver": "None",
       "os": "Linux",
       "database_os": "Linux",
-      "display_name": "Vibe.D",
+      "display_name": "vibe.d/MongoDB",
```
Comment (msmith-techempower) [inline frameworks/D/vibed/benchmark_config.json]: Drop the "/MongoDB" - that is displayed in a different way on the results website.

**Comment by msmith-techempower** [comment]
@s-ludwig Please rebase from our latest master for Travis-CI to test.

**Comment by s-ludwig** [comment]
Fixed and rebased. Now got failures in unrelated tests.

**Comment by s-ludwig** [comment]
Fixed and rebased (just got unrelated test failures now).

**Comment by msmith-techempower** [comment]
Travis-CI confirms that vibed is passing; LGTM!

**Comment by dejlek** [comment]
Well, it may pass the Travis CI there, but vibed fails at every benchmark except the plain one it seems...

**Comment by s-ludwig** [comment]
The current benchmark round just doesn't yet contain the fixed results.                                                                                                                                                                                                                                                                                                                                         Gesendet von meinem BlackBerry 10-Smartphone.                                                                                                                                                                                                                Von: Dejan LekicGesendet: Dienstag, 29. Dezember 2015 17:33An: TechEmpower/FrameworkBenchmarksAntwort an: TechEmpower/FrameworkBenchmarksCc: Sönke LudwigBetreff: Re: [FrameworkBenchmarks] Fixup vibe.d test project (#1771)Well, it may pass the Travis CI there, but vibed fails at every benchmark except the plain one it seems...

—Reply to this email directly or view it on GitHub.

**Pull request merged**